test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -146,7 +147,7 @@ public void EarlyStoppingTest()
         /// <summary>
         /// Multiclass Logistic Regression test.
         /// </summary>
-        [Fact]
+        [NotAppleSiliconFactAttribute("Temporarily skipping while helix issues are resolved. Tracked in issue #5845")]
```

Comment:
This is unfortunate that we need to disable these tests. But I guess it may be the best option for now, if we need to get this change in soon.

Reply:
So I disabled them because I needed to modify the logic for looking up the baseline output. But, that turned out to be much easier than I was initially expecting. I have the logic modified already and I am in the process of getting new baselines set up for these tests. Should have the changes pushed soon.

Reply:
Alright, the modified logic to look in the `osx-arm64` folder for baseline files has been pushed, along with all the new baseline files.

Reply:
So the NotAppleSiliconFactAttribute has been removed from all the tests in this file.